newsfragments/2672.feature.rst

```diff
@@ -0,0 +1 @@
+Whitespaces in character nicknames are now implicitly replaced with an underscore ("_").
```

Comment:
Is this a "feature" or more like "misc"?

Reply:
Oh, already merged, ok.

Reply:
Wow - must've submitted a review at the same exact moment!

Reply:
heh, it happens